### DISCUSSION TITLE
Help improve documentation.

### DISCUSSION THREAD

**Discussion by insertish** [stoatchat/discussions#60] [General]
Our documentation is not completely finished yet, if you find anything that needs to be improved, feel free to mention it here.

**Comment by Andre601** [comment]
The markdown reference has this raw syntax for the quotes:

```
> > If you change the way you look at things, the things you look at change.
> > — Wayne Dyer
```
while the result itself is shown as this:
> > If you change the way you look at things, the things you look at change.
>
> — Wayne Dyer

I assume that's a typo?

(Also, unrelated, but if possible can you make those example boxes a dark grey instead of bright white? It hurts in the eyes, even when using dark mode)

**Reply by DeMoN100** [reply]
Step by step pictures & video on how to setup a self hosted instance. Running it in localhost, development mode, & deploying a live third party instance. All step by step instructions with video & pictures.

**Comment by Andre601** [comment]
I was thinking: Wouldn't it be somewhat better to use tabs for Markdown example and result, rather than what it is right now?

The examples aren't really consistent too. At the start you have tables to display the raw and final markdown, later you have admonition boxes with the markdown and results... There isn't any consistency here.

With tabs you could have the raw in one and the result in another tab. You would also be able to group tabs, making them all switch when one is clicked.

If wanted am I up for implementing this.

**Comment by Monechk** [comment]
Здравствуйте! Добавьте пожалуйста возможность крепить важные сообщения в беседах, это очень важно и нужно, чтобы не потерять, и ещё создавать в каналах "#" подветви, которые имеют обсуждения внутри определённых сообщений. Это тоже очень нужная функция. Более пока ничего не волнует

**Reply by JoCat** [reply]
Зачем ты написал это в эту ветку? Это не по теме здесь.
